lib/mocha/stubbed_method.rb

```diff
@@ -108,10 +108,6 @@ def remove_original_method_from_stubbee
       original_method_owner.send(:remove_method, method_name)
     end
 
-    def use_prepended_module_for_stub_method?
-      RUBY_V2_PLUS
```

Comment:
Minor: I meant change the definition of `#use_prepended_module_for_stub_method?` to return `true` rather than replace all the call-sites with `true`, but this isn't a big deal.

Reply:
Ah I did that initially, but it essentially creates a few dead code paths so I figured I could take it all the way.